### PR TITLE
Change original array prototype lesson to conintued.  Added intro/jig…

### PR DESCRIPTION
**Pull request by Kalikoze**
…saw lesson for array prototype methods.

### CODE REVIEW

File: lessons/module-1/intro-to-array-prototype-methods.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+---
+title: Intro to Array Prototype Methods
+tags: JavaScript, array, prototype, introduction
+length: 150
```
Comment (damwhit) [inline lessons/module-1/intro-to-array-prototype-methods.md]: An hour and a half - should be 90?

File: lessons/index.html [hunk 2]
```diff
@@ -51,7 +51,8 @@ <h3>Javascript</h3>
     <li class="lesson"><a href="https://github.com/turingschool-examples/debugger-intro">Debugging Javascript</a></li>
     <li class="lesson"><a href="module-1/event-bubbling-and-delegation.html">Event Bubbling and Delegation</a></li>
     <li class="lesson"><a href="module-1/introduction-to-testing-javascript.html">Introduction to Unit Testing JavaScript</a></li>
-    <li class="lesson"><a href="module-1/array-prototype-methods-intro">Array Prototype Methods</a></li>
+    <li class="lesson"><a href="module-1/intro-to-array-prototype-methods.html">Intro to Array Prototype Methods</a></li>
+    <li class="lesson"><a href="module-1/array-prototype-methods-continued">Array Prototype Methods Continued</a></li>
```
Comment (damwhit) [inline lessons/index.html]: Could be good to call it 'array-prototype-iteration-methods'? That way they're building better vocabulary.

Comment (damwhit) [reply lessons/index.html]: Also, are you planning on streamlining that and pulling out some of the less essential pieces? (ie. lexicographical sorting)?

Comment (Kalikoze) [reply lessons/index.html]: Good call with the name.  Thanks!  Yes, I am planning on streamlining the iteration methods so it's a bit shorter and cutting out extras like the lexicographical sorting.

**Pull request merged**